core/templates/dev/head/pages/exploration_editor/editor_tab/customize_interaction_modal_directive.html

```diff
@@ -29,10 +29,20 @@ <h3 ng-if="stateInteractionIdService.displayed && (customizationModalReopened ||
 
   <form ng-if="stateInteractionIdService.displayed && hasCustomizationArgs" name="form.schemaForm" class="protractor-test-interaction-editor">
     <div ng-repeat="customizationArgSpec in customizationArgSpecs track by $index">
+      <div ng-if="customizationArgSpec.schema.type !== 'bool'">
       <div class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
-      <div>
-        <schema-based-editor local-value="stateCustomizationArgsService.displayed[customizationArgSpec.name].value" schema="customizationArgSpec.schema">
-        </schema-based-editor>
+        <div>
+          <schema-based-editor local-value="stateCustomizationArgsService.displayed[customizationArgSpec.name].value" schema="customizationArgSpec.schema">
+          </schema-based-editor>
+          <div style="margin-bottom: 15px;"></div>
+        </div>
+      </div>
+      <div ng-if="customizationArgSpec.schema.type === 'bool'" class="oppia-interaction-customization-label">
+        <div style="float:left;margin-right:4px;">
```

Comment:
Use a class tailored to this directive. Look at other directives which have style tags at the bottom of the directive, you can follow a similar pattern. See e.g. #4715.

Also see if you can apply some of this styling to the schema-based-editor tag directly. Having all these extra divs is kind of messy. (ng-class may be useful for applying conditional classes.)